decidim-dev/lib/decidim/dev/test/rspec_support/webmock.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 require "webmock/rspec"
-WebMock.allow_net_connect!
+
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
Is this intended? I don't see this on the PR description, that's why I ask

Reply:
Oh, yes, sorry. This was part of the debugging process but I left it there since it adds a bit of resilience against unintentionally slowing down the test suite when adding an external API.

Reply:
👍